packages/instant/src/components/animations/slide_up_and_down_animation.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+
+import { keyframes, styled } from '../../style/theme';
+
+const slideKeyframeGenerator = (fromY: string, toY: string) => keyframes`
+    from {
+        position: relative;
+        top: ${fromY}
+    }
+
+    to {
+        position: relative;
+        top: ${toY}
+    }
+`;
+
+export const SlideAnimation = styled.div<{ keyframes: string; animationType: string; animationDirection?: string }>`
+    animation-name: ${props => props.keyframes};
+    animation-duration: 0.3s;
+    animation-timing-function: ${props => props.animationType};
+    animation-delay: 0s;
+    animation-iteration-count: 1;
+    animation-fill-mode: ${props => props.animationDirection || 'none'};
+    position: relative;
+    z-index: -1;
```

Comment:
Felt hacky putting this here, but I needed it for the slide to come up from behind the instant widget.  I could make this a prop but felt a little messy having to pass it around.

Reply:
I think we should create a new stacking context for the container anyway since we do need to guarantee that it is above the page of the integrator.
ie. I think this z-index should be positive, and the container should simply have a higher index. 
We should probably add `position: relative` to the top level container and give it a z-index of 2, and the error one of 1. Also we should be keeping track of z-indexes in a central place.